apps/prairielearn/src/tests/questionSharing.test.ts

```diff
@@ -1,38 +1,39 @@
 import { assert } from 'chai';
 import { step } from 'mocha-steps';
-import { config } from '../lib/config';
+import * as fs from 'node:fs/promises';
+import * as path from 'node:path';
 import fetch from 'node-fetch';
-import { fetchCheerio } from './helperClient';
-import * as helperServer from './helperServer';
 import * as sqldb from '@prairielearn/postgres';
-const sql = sqldb.loadSqlEquiv(__filename);
-import { features } from '../lib/features/index';
-import { EXAMPLE_COURSE_PATH, TEST_COURSE_PATH } from '../lib/paths';
 
+import { config } from '../lib/config';
+import { Course } from '../lib/db-types';
+import { features } from '../lib/features/index';
+import { selectCourseById } from '../models/course';
 import * as syncFromDisk from '../sync/syncFromDisk';
-
+import { fetchCheerio } from './helperClient';
+import * as helperServer from './helperServer';
 import { makeMockLogger } from './mockLogger';
+import * as syncUtil from './sync/util';
+
+const sql = sqldb.loadSqlEquiv(__filename);
 const { logger } = makeMockLogger();
```

Comment:
why all these changes? are we using some new import sorting method or something?

Reply:
We don't yet have a lint rule to enforce this (so I don't say anything on PRs), but yes, I try to keep first-party and third-party inputs separate, and I opportunistically clean that up in existing files when I can.